app/controllers/applications_controller.rb

```diff
@@ -69,9 +69,13 @@ def deploy
     @release_tag = params[:tag]
 
     if application_has_dashboard?(@application.shortname)
-      @staging_dashboard_url = dashboard_url('grafana.staging.publishing.service.gov.uk', @application.shortname)
-      @production_dashboard_url = dashboard_url('grafana.publishing.service.gov.uk', @application.shortname)
-      @aws_staging_dashboard_url = dashboard_url('grafana.staging.govuk.digital', @application.shortname)
+      if @application.on_aws
```

Comment:
Minor: you can use `application.on_aws?` everywhere instead of `application.on_aws` because this is a boolean.

Reply:
Oh nice! Done in 7816a57de441287f5d2d8bb15c309c1ec336e010

Reply:
👍